pkgs/data/icons/posy-cursors/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, stdenvNoCC, fetchgit }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "posy-cursors";
+  version = "1.6";
+
+  src = fetchgit {
+    url = "https://github.com/simtrami/posy-improved-cursor-linux";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-i0N/QB5uzqHapMCDl6h6PWPJ4GOAyB1ds9qlqmZacLY=";
```

Comment:
Can you use `hash` ?

Reply:
Do you mean a commit hash?

Reply:
I mean `hash = "sha256-i0N/QB5uzqHapMCDl6h6PWPJ4GOAyB1ds9qlqmZacLY=";`